store/interfaces.json

```diff
@@ -34,6 +34,14 @@
       "$ref": "app:vtex.product-price#/definitions/Installments"
     }
   },
+  "product-installments.list": {
+    "component": "InstallmentsList",
+    "composition": "children",
+    "overrideContentRule": "self",
```

Comment:
I think we should go for not extending interfaces anymore. It doesn't make much sense.

I would change this interface name to `product-installments-list`

And remove the need for this:
```suggestion
```